application/models/team.py

```diff
@@ -0,0 +1,63 @@
+from application.errors import NotFound, NotInitialized
+from flask import current_app
+from py2neo import Graph, Node
+import uuid
+
+
+class Team:
+
+    @classmethod
+    def all(cls):
```

Comment:
What is `cls`?

Reply:
All class methods in python do have cls (class) as the first parameter that is always implicitly passed to the method.
normal instance methods have self passed instead.